src/ui/common/src/components/pages/check/id/index.tsx

```diff
@@ -0,0 +1,301 @@
+        data: mockCheckArtifactData
+    };
+
+    // Function to get the numerical value of the metric output
+    // TODO: Use this inside of the accordion component below.
+    // NOTE: This code is shared with the metric details page, perhaps we should make this into a hook or component.
+    const operatorOutputsList = operator.outputs.map((artifactId) => {
+        const artifactResult = (workflowDagResultWithLoadingStatus.result
+            ?.artifacts ?? {})[artifactId];
+        if (!artifactResult) {
+            return null;
+        }
+
+        console.log('operator output artifactResult: ', artifactResult);
+
+        if (
+            !artifactResult.result ||
+            artifactResult.result.content_serialized === undefined
+        ) {
+            // Link to appropriate artifact details page
+            // Show tableIcon here as part of the link.
+            return (
+                <Box key={artifactId}>
+                    <Link
+                        to={`${getPathPrefix()}/workflow/${workflowId}/result/${workflowDagResultId}/artifact/${artifactId}`}
+                        component={RouterLink as any}
+                        sx={{ marginLeft: '16px' }}
+                        underline="none"
+                    >
+                        {artifactResult.name}
+                    </Link>
+                </Box>
+            );
+        }
+
+        return (
+            <Box key={artifactId} display="flex">
+                {/* <Typography variant="body1" marginRight="24px">
+                    {artifactResult.result.content_serialized}
+                </Typography> */}
+                <CheckTableItem checkValue={artifactResult.result.content_serialized} />
+                <Link
+                    to={`${getPathPrefix()}/workflow/${workflowId}/result/${workflowDagResultId}/artifact/${artifactId}`}
+                    component={RouterLink as any}
+                    sx={{ marginLeft: '16px' }}
+                    underline="none"
+                >
+                    {artifactResult.name}
+                </Link>
+            </Box>
+        );
+    });
+
+    return (
+        <Layout user={user}>
+            <Box width={'800px'}>
+                <Box width="100%">
+                    <Box width="100%">
+                        <DetailsPageHeader name={operator?.name} />
+                        {operator?.description && (
+                            <Typography variant="body1">{operator.description}</Typography>
+                        )}
+                    </Box>
+                </Box>
+
+                <Box width="100%" marginTop="32px">
+                    <Typography variant="h5" marginBottom="8px">Recent Results</Typography>
+                    <CheckHistory historyWithLoadingStatus={artifactHistoryWithLoadingStatus} checkLevel={operator?.spec?.check?.level} />
+                </Box>
+                {/* commenting out metrics for now as we figure out what to do with them */}
```

Comment:
TODO: Find out if this is going to be something to support. But probably going to end up removing for now and bring back if so.